docs/source/changelog.rst

```diff
@@ -6,7 +6,7 @@ Jupyter notebook changelog
 A summary of changes in the Jupyter notebook.
 For more detailed information, see `GitHub <https://github.com/jupyter/notebook>`__.
 
-.. _release-4.1.x:
```

Comment:
Do we want a label for every release, or just for every feature release? Should we leave a 4.1.x label in as well so we can always cross-link to that section?